tox.ini

```diff
@@ -34,16 +34,16 @@ deps = coverage
 skip_install = True
 changedir = {toxworkdir}
 setenv = COVERAGE_FILE=.coverage
-commands = coverage erase
-           coverage combine
-           coverage report --rcfile={toxinidir}/tox.ini
-           coverage xml
+commands = python -m coverage erase
```

Comment:
Why change these?

Reply:
Because they failed on Windows without.

Reply:
I don't think this should be necessary, but I also don't think it hurts as long as it works, so I'm fine with it.

Reply:
I couldn't find the log anymore, but I think it was necessary for the steps to work on pypy3 but not for pypy2 for some reason. I just assumed it being one of those Windows things.